lib/presentation/themes/theme.dart

```diff
@@ -139,36 +139,36 @@ abstract class LocalTheme {
         fontFamily: primaryFontBlack,
         fontSize: titleXLFontSize,
         height: titleXLHeight,
-        letterSpacing: -0.07,
+        letterSpacing: -.07,
```

Comment:
These changes are added to remove the "Double literal shouldn't begin with '.'." warning it generates. Next step is to move them out to remove the "Avoid using magic numbers. Extract them to named constants or variables." I think we should leave them. WDYT

Reply:
Got it! I'm gonna look for them in the project and change them

Reply:
Done